dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/LazyLoadingTest.java

```diff
@@ -70,21 +70,23 @@ public void run(TestConfiguration configuration, Environment environment) throws
 
             environment.jersey().register(new UnitOfWorkApplicationListener("hr-db", sessionFactory));
             environment.jersey().register(new DogResource(new DogDAO(sessionFactory)));
+            environment.jersey().register(new PersistenceExceptionMapper());
             environment.jersey().register(new ConstraintViolationExceptionMapper());
         }
 
         private void initDatabase(SessionFactory sessionFactory) {
             try (Session session = sessionFactory.openSession()) {
-                session.createSQLQuery(
+                session.beginTransaction();
```

Comment:
The same comment about the transaction. I think it would be more clear to commit it manually.